### DISCUSSION TITLE
Override Combobox's filter method

### DISCUSSION THREAD

**Discussion by jtruffot** [vmware-clarity/ng-clarity#1209] [Q&A] [answered]
Currently, the combobox performs case-insensitive filtering. Would it be possible to override this behavior to allow accent-insensitive searches or something more complex?

![image](https://github.com/vmware-clarity/ng-clarity/assets/65965919/763df8e6-1ca6-4584-b334-9b1d3967ade5)

I managed to implement a [solution](https://stackblitz.com/edit/clarity-light-theme-clr17-wytsxt?file=src%2Fapp%2Foptions-items-matcher.directive.ts) but as it stands, the [ClrOptionItems](https://github.com/vmware-clarity/ng-clarity/blob/main/projects/angular/src/forms/combobox/option-items.directive.ts) directive seems impossible to extend properly.

**Comment by kevinbuhmann** [comment, marked as answer]
Thanks for this report! I made the combobox search diacritic-insensitive in #1210. This change will be released in 15.x, 16.x, and 17.x.
